libr/util/idpool.c

```diff
@@ -163,6 +162,54 @@ R_API void* r_id_storage_get(RIDStorage* storage, ut32 id) {
 	return storage->data[id];
 }
 
+R_API bool r_id_storage_get_lowest(RIDStorage *storage, ut32 *id) {
+	r_return_val_if_fail (storage, false);
+	size_t i;
+	for (i = 0; i < storage->size && !storage->data[i]; i++);
+	*id = i;
+	return i < storage->size;
+}
+
+R_API bool r_id_storage_get_highest(RIDStorage *storage, ut32 *id) {
+	r_return_val_if_fail (storage, false);
+	size_t i = 0;
+	if (storage->size > 0) {
+		for (i = storage->size - 1; !storage->data[i] && i > 0; i--);
+		*id = i;
+		return storage->data[i];  // != NULL;
```

Comment:
just put != NULL in the code instead of the comment... if it is not clear enough, it's worth putting it in the code, as it gets compiled.